docs/user_guide/migrating_versions/migrating_to_v0_3.rst

```diff
@@ -0,0 +1,38 @@
+Migrating to v0.3 from v0.2
+===========================
+
+.. raw:: html
```

Comment:
Sorry, one section left to remove. Everything here from line 4-8 isn't needed anymore.

Reply:
Ohh sorry. Thanks I'll change that now